### PR TITLE
Exporting the client struct from mock.go

### PR DESCRIPTION
**Pull request by mensurm**
This PR is in connection to the discussion here
https://dev.azure.com/skfdc/REP-SW/_git/users/pullrequest/33270?_a=files&discussionId=262086&path=%2Fbackend%2Fbusinesslogic%2Fauthorizersync%2Fsync_test.go

I would like to export the client struct so it can be directly accessed, in the same way how it was done for example on the users repo, with the `DataLayer` struct or `ClientMock` in the hierarchy package. 

Please have a look.

### CODE REVIEW

**Review by garoyeri — CHANGES_REQUESTED** [review]
If you're exporting the client type on `/services/authorize/mock`, you should do the same for `/v2/services/authorize/mock` to make them consistent.

**Comment by mensurm** [comment]
@garoyeri Changed

File: .travis.yml [hunk 1]
```diff
@@ -9,10 +9,12 @@ install:
   - $(go env GOPATH)/bin/golangci-lint --version
 script:
   - $(go env GOPATH)/bin/golangci-lint run --disable wsl ./...
+  - go clean
   - go build ./...
-  - go test ./... --short
+  - go test ./... --short -count=1
```
Comment (garoyeri) [inline .travis.yml:14]: What does this change actually do? The explanation of this must show up in the squashed message after the merge so we know what the purpose here is.

Comment (mensurm) [reply .travis.yml:14]: It's disabling the caching of test results

**Review by kulbhushan-skf — APPROVED** [review]
Changes looks good to me.

**Comment by mensurm** [comment]
Exporting authorize mock struct. Disabling caching of test results.

**Pull request merged**